src/main/java/nextstep/qna/service/QnAService.java

```diff
@@ -25,7 +27,9 @@ public class QnAService {
     @Transactional
     public void deleteQuestion(NsUser loginUser, long questionId) throws CannotDeleteException {
         Question question = questionRepository.findById(questionId).orElseThrow(NotFoundException::new);
+        question.delete(loginUser);
+        List<DeleteHistory> deleteHistories = question.toDeleteHistories();
```

Comment:
구조상 삭제 하지 않아도 삭제 이력을 생성할 수 있게 됩니다.
삭제와 동시에 삭제 이력을 생성하는 방향도 고려해 보시면 좋을 것 같아요.
```suggestion
        List<DeleteHistory> deleteHistories = question.delete(loginUser);
```